project/pages/LineSelected/index.jsx

```diff
@@ -0,0 +1,159 @@
+import "leaflet/dist/leaflet.css"
+import { useState } from "react";
+import { Circle, MapContainer, Polyline, Popup, TileLayer } from "react-leaflet";
+
+import { CgArrowsExchangeAltV } from "react-icons/cg";
+import { FaArrowDown } from "react-icons/fa";
+import { HiUserGroup } from "react-icons/hi2";
+import { FaUserGroup } from "react-icons/fa6";
+import { IoPersonSharp } from "react-icons/io5";
+import { useParams } from "react-router-dom";
+
+import { specificPolyline, allStationsCoords, specificStationName, routeName, stopsNameAndCrowding, } from "../../utils";
+import { useSelector } from "react-redux";
+
+
+// visualize Map components
+
+function allStationsVisualised(lineName, transportData) {
+
+    return (allStationsCoords(lineName, transportData).map((coords, index) => {
+        return (
+
+            <Circle
+                center={coords}
+                pathOptions={{ color: "black", fillColor: "white", fillOpacity: 1 }}
+                radius={20}
+                key={index}
+            >
+                <Popup
+                >
+                    {specificStationName(lineName, index, transportData)}
+                </Popup>
+            </Circle>
+
+        )
+
+    }))
+}
+
+// visualize Table components
+
+function visualizeCrowding(input) {
+
+    if (input === 'NaN') {
+        return 'NaN'
+    }
+
+    let number = Math.ceil(Number(input))
+
+    if (number > 5) {
+        return <HiUserGroup className="text-red-600" size={20} />
+    } else if (number >= 0) {
+        return <FaUserGroup className="text-yellow-600" size={20} />
+    } else if (number < 0) {
+        return <IoPersonSharp className="text-green-600" size={20} />
+    }
+}
+
+
+
+const Line = () => {
+
+    // Redux state
+    const transportData = useSelector((state) => state.data.data)
+    const dataStatus = useSelector(state => state.data.status)
+    const errorData = useSelector(state => state.data.error)
+
+    const colors = useSelector((state) => state.colors.colors)
+    const colorsStatus = useSelector(state => state.colors.status)
+    const errorColors = useSelector(state => state.colors.error)
+
+    // Easier toggle with Boolean based on being or not being route A
+    const [routeA, setRouteA] = useState(true)
```

Comment:
Boolean variables should be names in the form of a question.
```suggestion
    const [isRouteA, setIsRouteA] = useState(true)
```